.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,67 @@
+name: Bug Report
+description: "Create a report to help us improve."
```

Comment:
maybe add something along the lines of "we'd love if you'd first take a look through our open issues to make sure the issue isn't already filed"